src/reducer.js

```diff
@@ -11,43 +11,43 @@ function getHandlers({customHandlers, types, resourceName, pluralForm, allowedOp
   const {create, readOne, readMany, update, del} = allowedOperations;
 
   const createHandlers = create ? {
-    [`CREATE_${capitalResourceName}`]: defaultReducers.create,
-    [`CREATE_${capitalResourceName}_FAILURE`]: defaultReducers.createFailure,
-    [`CREATE_${capitalResourceName}_SUCCESS`]: defaultReducers.createSuccess,
-    [`CREATE_${capitalResourceName}_ABORTED`]: defaultReducers.createAborted,
-    [`CREATE_${capitalResourceName}_RESET_RESOLUTION`]: defaultReducers.createResetResolution
+    [`CREATE_${capitalResourceName}`]: defaultReducers.create.bind(null, idAttr),
```

Comment:
`.bind` has become more and more something I avoid using. I think it's ok here, just wanted to Give My Opinion™ that I'd rather see this created with an initialization / factory style, or using a curry ala `(...args) => defaultReducers.create(idAttr, ...args)`